test/kyberUniswapReserve.js

```diff
@@ -1,4 +1,5 @@
-const web3 = require("web3");
+const Web3 = require('web3');
+const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
```

Comment:
don't follow why web3 changes?
can't we keep it as is?

Reply:
Nope, I'm not sure why I'll get `web3.toWei is not a function` for all the tests if I do.